admin/base/templatetags/base_extras.py

```diff
@@ -0,0 +1,8 @@
+from django.core.urlresolvers import reverse
+from django.utils.http import urlencode
+
+
+def reverse_qs(view, urlconf=None, args=None, kwargs=None, current_app=None, query_kwargs=None):
```

Comment:
This is useful, but why is this in `templatetags` ? Would it make more sense in `admin.base.utils`?

Also minor: `query_kwargs` is a bit of a misnomer because you don't pass them as keyword arguments. How about just `query`?